### PR TITLE
3.0.0

### PR DESCRIPTION
**Pull request by brewster1134**

### CODE REVIEW

File: src/bumper-dom-jquery.coffee [hunk 1]
```diff
@@ -35,12 +36,13 @@
         $element = $rootEl.closest("#{splitArray[0]}")
 
         # find first matching elemnt anywhere in the dom
-        $element = $("#{splitArray[0]}").first() unless $element.length
+        if restrictToParents == false && !$element.length
```
Comment (brewster1134) [inline src/bumper-dom-jquery.coffee]: @mmoriar1 this is the option we had talked about

File: src/bumper-responsive-image.coffee [hunk 2]
```diff
@@ -62,6 +95,28 @@
...
+
+      # Log warning if no url is defined
+      throw new Error "data-bumper-responsive-image-url[-#{breakpoint}] is not set." unless url
+
+      # combine params if they are found
+      fullUrl = if params then "#{url}?#{params}" else url
+
+      # detect if inteprolation is needed
+      if window.Bumper.Dom? && fullUrl.indexOf('{') >= 0
+        fullUrl = window.Bumper.Dom.interpolateElementAttrs fullUrl, el, @options.restrictToParents
```
Comment (brewster1134) [inline src/bumper-responsive-image.coffee]: @mmoriar1 and this i where its called.  you can set it via `window.Bumper.Responsive.Image. restrictToParents = true`

Comment (mmoriar1) [inline src/bumper-responsive-image.coffee]: set it via that or on bumper init right?

Comment (brewster1134) [inline src/bumper-responsive-image.coffee]: when you config and initialize bumper, you would set this.  for audi, we would set this in our `responsive-config.js`

**Comment by mmoriar1** [comment]
looks great :+1:

**Pull request merged**